src/main/java/com/danielagapov/spawn/Mappers/EventMapper.java

```diff
@@ -15,26 +16,23 @@ public static EventDTO toDTO(Event entity) {
                 entity.getTitle(),
                 entity.getStartTime(),
                 entity.getEndTime(),
-                entity.getLocation(),
+                entity.getLocation() != null ? LocationMapper.toDTO(entity.getLocation()) : null, // Map Location to LocationDTO
```

Comment:
Makes sense 👍 Let's stick to this syntax for null-checking in the future, as well, to stay consistent.